GOVERNANCE.md

```diff
@@ -198,3 +198,19 @@ placed on the www server for access to the "dist" user. The private key should
 be kept very secure and not shared with anyone. Any actual or suspected
 compromise of the private key should be reported immediately and the key should
 be entirely removed from use.
+
+## Offboarding releasers
+
+*Note*: This section is specifically concerning removing members from the [@nodejs/releasers](https://github.com/orgs/nodejs/teams/releasers) team, not removing members from the overall Release WG.
+
+Releasers have access to critical infrastructure in the project - this elevated access must be restricted to active releasers. Members of the releasers team should be offboarded when they no longer intend to prepare releases. As a guideline, offboarding should be considered if a releaser has not prepared a release in the past 18 months.
```

Comment:
I welcome thoughts/feedback on the specific '18 months'. IMO anywhere from 12months - 24months seems reasonable.

Reply:
12 months would be good with me, people can always be re-added.

Reply:
+1 on 12 months

Reply:
I've updated the guidance to 12 months. Will land tomorrow if there are no concerns.